doc/api/n-api.md

```diff
@@ -3207,13 +3207,16 @@ added: v8.0.0
 ```C
 NAPI_EXTERN
 napi_status napi_create_async_work(napi_env env,
+                                   const char* async_resource_name,
                                    napi_async_execute_callback execute,
                                    napi_async_complete_callback complete,
                                    void* data,
                                    napi_async_work* result);
 ```
 
 - `[in] env`: The environment that the API is invoked under.
+- `[in] async_resource_name`: An identifier for the kind of resource that is
```

Comment:
Since this is arbitrary, maybe it needs another sentence. Something like:
```md
The value is provided by the embeded and should be representative of the type of async work being performed.
```